lighthub/utils.cpp

```diff
@@ -100,11 +100,13 @@ extern char _end;
 extern "C" char *sbrk(int i);
 
 unsigned long freeRam() {
+#ifndef DISABLE_FREERAM_PRINT
     char *heapend = sbrk(0);
     register char *stack_ptr asm( "sp" );
     struct mallinfo mi = mallinfo();
-
     return stack_ptr - heapend + mi.fordblks;
+#endif
+    return 7777;
```

Comment:
Bad idea. freeram() used not just for printing but to prevent stack overflow in mqtt callback